.gitignore

```diff
@@ -20,3 +20,4 @@
 
 # Ignore application configuration
 /config/application.yml
+
```

Comment:
This is an unnecessary change (see our checklist). Removing it may help avoid merge conflicts in more complicated cases, and it's good practice to avoid this kind of thing.